introduction/documents.md

```diff
@@ -0,0 +1,61 @@
+# *Documents*
+
+ - A *Document* can be a .pdf, .jpeg or a .png file along with some meta information.
+    
+Lucidtech delivers services that helps you control and automate the flow
+of your documents, and a *Document* is therefore an important concept, and in this 
+introduction you will see how a *Document* can be created, controlled and used together with 
+*Batches*, *Consents*, *Predictions*, and *Models*.
+
+# Creating a *Document*
+The simplest way to create a *Document* is to just specify a path;
```

Comment:
'to just specify a path' -> 'to use the CLI and use the path of the PDF or JPEG that you would like to upload' Would something along these lines be clearer?